qiskit_braket_provider/providers/__init__.py

```diff
@@ -17,9 +17,9 @@
     AWSBraketBackend
     BraketLocalBackend
     AWSBraketProvider
-    AWSBraketJob
+    AmazonBraketTask
 """
 
 from .braket_backend import AWSBraketBackend, BraketLocalBackend
 from .braket_provider import AWSBraketProvider
-from .braket_job import AWSBraketJob
```

Comment:
This might be a breaking change for code which does something like `from qiskit_braket_provider.providers import AWSBraketJob`

Reply:
this has now been resolved with the latest commit https://github.com/qiskit-community/qiskit-braket-provider/pull/105/commits/abb7925956ff8970d4f3399629be30011655ddb6